### PR TITLE
Adding info to the banner

### PR DESCRIPTION
**Pull request by sfshaza2**
Andrew asked me to add this link. He might also want it on dart.dev?

### CODE REVIEW

**Comment by RedBrogdon** [comment]
@kwalrath, Dart questions, particularly about null safety, would also be very welcome!

File: src/_includes/banner.html [hunk 1]
```diff
@@ -4,6 +4,8 @@
   <div class="site-banner site-banner--default" role="alert">
     {% comment %} For headings use <h3> elements. {% endcomment -%}
     <a href="https://g.co/flutterday">Flutter Day is June 25th. Register now!</a><br>
+    Tweet your questions for #AskFlutter. For more info,
+    see the <a href="https://twitter.com/FlutterDev/status/1272544935426678785">tweet!</a>.
```
Comment (kwalrath) [inline src/_includes/banner.html:8]: There's an extra `.`, this is kind of long, and the link is unclear. How about maybe:

```suggestion
    Tweet your questions for #AskFlutter.
    <a href="https://twitter.com/FlutterDev/status/1272544935426678785">Details here!</a>
```

Comment (kwalrath) [reply src/_includes/banner.html:8]: Also... Is there a good reason to keep the commented-out text?

**Comment by sfshaza2** [comment]
Yes, please do not delete. I keep that for reference.

**Comment by sfshaza2** [comment]
LGTM

**Pull request merged**